src/deephaven/plugin/object.py

```diff
@@ -17,46 +17,84 @@ def index(self) -> int:
         May be used in the output stream to refer to the reference from the client."""
         return self._index
 
-    @property
-    def type(self) -> Optional[str]:
-        """The type."""
-        return self._type
-
 
 class Exporter(abc.ABC):
-    """The interface for creating new references during ObjectType.to_bytes."""
+    """The interface for creating new references during FetchOnlyObjectBase.to_bytes."""
+
+    @abc.abstractmethod
+    def reference(self, obj: object) -> Reference:
+        """Creates a reference for an object. Each reference """
+        pass
+
+
+class MessageStream(abc.ABC):
+    """A stream of messages, either sent from server to client or client to server. ObjectType implementations
+    provide an instance of this interface for each incoming stream to invoke as messages arrive, and will
+    likewise be given an instance of this interface to be able to send messages to the client.
+    """
+    def __init__(self):
```

Comment:
```suggestion
    """
    
    def __init__(self):
```
I think this space is PEP8 standard

Reply:
My IDE doesn't agree, and my read of PEP8 also disagrees: 

> Surround top-level function and class definitions with two blank lines.
>
> Method definitions inside a class are surrounded by a single blank line.
>
> Extra blank lines may be used (sparingly) to separate groups of related functions. Blank lines may be omitted between a bunch of related one-liners (e.g. a set of dummy implementations).

https://peps.python.org/pep-0008/#blank-lines

This is a method inside a class, so just one line, correct?

Reply:
The pydoc for the class is touching the constructor function, which has no explicit pydocs, so it is not covered by the excerpt included here.  It is closest to #2 in the list and is what I think the rest of our py has done.

Reply:
Either way, not critical.  An eventual auto-formatter will fix it in a consistent way everywhere.

Reply:
I misread, you're correct.